Formula/x265.rb

```diff
@@ -20,6 +20,8 @@ class X265 < Formula
   depends_on "cmake" => :build
   depends_on "nasm" => :build if Hardware::CPU.intel?
 
+  patch :DATA if build.head?
```

Comment:
We don't carry patches for head builds; sorry!

Reply:
I am not sure whether I get it or not.
Currently the `brew install x265 --HEAD` is broken on Apple Silicon, and this patch will fix it. Even that, homebrew won't accept the patch?

Reply:
That is correct. We usually just remove `head` when it's broken.